ast/Trees.cc

```diff
@@ -413,8 +413,13 @@ string MethodDef::toStringWithTabs(const core::GlobalState &gs, int tabs) const
     } else {
         buf << "def ";
     }
+    auto funcName = name.data(gs)->toString(gs);
+    buf << funcName;
     auto &data = this->symbol.dataAllowingNone(gs);
-    buf << name.data(gs)->toString(gs) << "<" << data->name.data(gs)->toString(gs) << ">";
+    auto symName = data->name.data(gs)->toString(gs);
+    if (funcName != symName) {
```

Comment:
Can you do the comparison at `NameRef`, rather than at `std::string`?

Reply:
nice idea!